token-lending/program/tests/helpers/mod.rs

```diff
@@ -520,14 +523,15 @@ impl TestLendingMarket {
         banks_client: &mut BanksClient,
         payer: &Keypair,
         args: LiquidateArgs<'_>,
-    ) {
+    ) -> Result<(), TransportError> {
```

Comment:
Could you use `TransactionError` instead? `TransportError` can be unwrapped to it